src/LocaleAwareValidatorInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+namespace Zend\Validator;
+
+interface LocaleAwareValidatorInterface
+{
+    /**
+     * Set locale used for validation.
+     *
+     * @param string $locale
+     */
+    public function setLocale($locale = null);
```

Comment:
I've made this an optional parameter, since zend-filter also made it optional and it makes it possible to pass `null`. Making it optional in zend-i18n validators shouldn't make it a BC.